tests/unittests/execution_store_unit_tests.py

```
@@ -420,3 +420,262 @@ def test_string_dataset_id_raises_exception(self):
+        # can't be satisfied by other tests importing the clone module
+        # directly (which would also trigger registration).
+        import subprocess
+        import sys
+
+        code = (
+            "import fiftyone\n"
+            "import fiftyone.core.dataset as d\n"
+            "names = [getattr(c, '__name__', '') for c in d._extras_cloners]\n"
+            "assert 'clone_execution_stores' in names, names\n"
+        )
+        subprocess.run([sys.executable, "-c", code], check=True)
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)